pages/models/generations/GENQEC/index.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: page
+title: GENQEC Synchronous Machine with Saturation
+tags: ["#197","Synchronous machine", "Rotor", "Stator", "PowerWorld", "Opensource", "GENROU", "GENSAL", "GENTPF", "GENTPJ", "GENQEC", "Saturation", ]
+date: 08/08/2024 
+last-updated: 08/08/2024
+id: #197
+authors: Carlos Alegre (eRoots)
+reviewers: Eduardo Prieto Araujo (UPC), Josep Fanals Batllori (eRoots)
+---
+
+## Context
+
+Saturation modelling in synchronous machines is key to perform transient stability studies. As described in the [GENTPF/GENTPJ](../GENTPJ/) models, a more accurate modeling of the saturation is achieved by defining saturation functions, which depend on the magnetic flux and the currents of the synchronous machine, to scale the reactances and the fluxes. In the GENTPJ model, this is only done for the subtransient reactances. Some issues were found with this model [[1]](#1), which resulted in the development of an improved model, the GENQEC model [[2]](#2), which is described in this page.
+
+## Model use, assumptions, validity domain and limitations
+
+As the GENTPF/GENTPJ model, it is used to perform transient stabililty of synchronous machines with the most complete modeling of saturation.
+
+The model assumptions are the following:
+
+* The three-phase system is balanced.
+* The air gap between the stator and rotor is non-uniform, but the relationships between the flux linkages and currents are independent of the position of the rotor when expressed in the *dq0* reference frame.
+* The magnetic circuit is considered to be non-linear, considering saturation effects, and can be modeled using different saturation functions.
+* The saturation effects are applied to all the reactances, but not at the transient and steady-state reactances.
+
+The GENQEC model is suitable for EMT as it models with detail the non-linearities and the fast dynamics of the machine. It will substitute the GENTPJ in December 2024 after the testing phase is finished, although it is already accepted by the WECC [[3]](#3).
+
+## Model Description
+
+### Parameters
+
+| Parameter | Description | Unit |
+| --- | --- | --- |
+| $$\omega_s$$ | Synchronous speed | rad/s |
+| $$R_s$$ | Stator resistance | $$\Omega$$ |
+| $$X_{dsat}$$ | Saturated Direct-axis synchronous reactance| $$\Omega$$ |
+| $$X'_{dsat}$$ | Saturated Direct-axis transient reactance | $$\Omega$$ |
+| $$X_{qsat}$$ | Saturated Quadrature-axis synchronous reactance | $$\Omega$$ |
+| $$X'_{qsat}$$ | Saturated Quadrature-axis transient reactance | $$\Omega$$ |
+| $$X''_{dsat}$$ | Saturated Direct-axis subtransient reactance | $$\Omega$$ |
+| $$X''_{qsat}$$ | Saturated Quadrature-axis subtransient reactance| $$\Omega$$ |
+| $$X''_{dsat}$$ | Saturated Direct-axis subtransient reactance with saturation considered| $$\Omega$$ |
```

Comment:
The parameter "$$X''_{dsat}$$" used is similar to the one mentioned above. Is it meant to be same?

Same with ""$$X''_{qsat}$$"

Reply:
It was duplicated, removed